M2/Macaulay2/editors/pygments/macaulay2.py.in

```diff
@@ -12,6 +12,8 @@ from pygments.token import Comment, Keyword, Name, String, Text
 
 __all__ = ['Macaulay2Lexer']
 
+@M2BANNER@
```

Comment:
What if you put it in the docstring, perhaps on line 4?

Reply:
Their test is pretty picky, and anything outside of the default copyright header in the docstring raises an error.  When I moved the banner up and ran the test script, I got:

```
pygments/lexers/macaulay2.py:0: no correct license info
pygments/lexers/macaulay2.py:0: no correct copyright info
```